setup.cfg

```diff
@@ -1,80 +1,38 @@
-[bdist_wheel]
```

Comment:
```universal = 1``` is wrong, dbxincluder is python 3 only.
Leaving the section empty is basically the same as removing it.